ac/ac-imgView/src/ActivityRunner.jsx

```diff
@@ -34,7 +34,6 @@ const ActivityPanel = ({
   webcam,
   setWebcam
 }) => {
-
```

Comment:
Try to check which files you are adding to git, instead of just adding all files. this is obviously not related to this PR.